{

  debug_mode: true,
  enable_partial_update: true,
  enable_mocap_switch: true,
  mahalanobis_d_sonar: 20,
  mahalanobis_d_rgbd_velocity: 50,
  mahalanobis_d_mocap_z: 20,
  mahalanobis_d_mocap_velocity: 20,
  mahalanobis_distance_mocap_xy_: 50,
  enable_measurements: false,
  mocap_override_channel: 6
}
